packages/@aws-cdk/aws-backup/README.md

```diff
@@ -218,3 +236,27 @@ const role = new iam.Role(this, 'Access Role', { assumedBy: new iam.ServicePrinc
 
 importedVault.grant(role, 'backup:StartBackupJob');
 ```
+
+## Backup selection IAM role policies
+
+When creating a new `BackupSelection`, if no user IAM role is provided, a new
+IAM role with a managed policy for backup will be created for the selection. 
+
+All policies, by default, include the `AWSBackupServiceRolePolicyForBackup` AWS
+managed policy. Adding the `allowRestores` flag adds the
+`AWSBackupServiceRolePolicyForRestores` managed policy.
+
+If you are creating a selection which includes an **S3 Bucket**, then the
+default service roles are not sufficient for backup nor restore operations. To
+enable S3 backups for a selection, include the `allowS3Backup` flag (the
+`AWSBackupServiceRolePolicyForS3Backup` managed policy). To enable S3 restores
+for a selection, include the `allowS3Restores` flag (the
+`AWSBackupServiceRolePolicyForS3Restore` managed policy). Both of these flags
+are `false` by default.
+
+If the generated IAM roles for a `BackupSelection` are insufficient, you can
+either include your own IAM role using the optional `role` field, or access the
+generated IAM role with the read only `role` property. If you want to have
+complete control over role policies, you should provide your own role with the
+`role` field and disable any automatic addition of managed policies with the
+`customRole` field.
```

Comment:
This is just me being extra, but if you find it in your heart to add the code samples going from one option to the other, that's be 🙌

Reply:
Happy to do that - not extra at all :rofl: